### PR TITLE
Packaging fixes

### PR DESCRIPTION
**Pull request by willkg**
* added setup.cfg with flake8 arguments
* bunch of cosmetic flake8 related fixes
* added jingo requirement to setup.py
* updated MANIFEST.in

### CODE REVIEW

**Comment by willkg** [comment]
@diox Can you skim this for issues? Most of it is cosmetic, but it also adds the jingo requirement to `setup.py`.

File: MANIFEST.in [hunk 1]
```diff
@@ -2,4 +2,9 @@ include LICENSE
 include CONTRIBUTORS
 include README.rst
 include CHANGES.rst
+include run_tests.py
+include requirements.txt
+include contribute.json
 include jingo_minify/bin/yuicompressor-2.4.7.jar
+
+recursive-include examples *.css *.py
```
Comment (diox) [inline MANIFEST.in]: Missing newline at EOF

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 100
+ignore =
+    # E731: do not assign a lambda expression
+    E731
```
Comment (diox) [inline setup.cfg]: I'd just fix those since there are only 2 instances left - you already did the most annoying work by fixing the weird indentation :)

**Review by diox — APPROVED** [review]
Looks good to me

**Comment by willkg** [comment]
Yeah... I looked at the two places and I could fix them, but I don't think it helps much. I think it's easier/better just to raise the max line length.

**Comment by willkg** [comment]
@diox After this lands, are there other changes you need? Do you want me to do a release?

**Comment by diox** [comment]
If you could make a release with those changes that'd be great yeah - I don't think I need other changes - thanks again!

**Comment by willkg** [comment]
@diox 0.7 released. If there are any issues, let me know.

**Pull request merged**